python/lsst/jointcal/jointcal.py

```diff
@@ -500,6 +499,40 @@ def _fit_astrometry(self, associations):
 
         return Astrometry(fit, model, sky_to_tan_projection)
 
+    def _prep_astrometry_constrainedPolyModel(self, associations, sky_to_tan_projection):
+        model = lsst.jointcal.ConstrainedPolyModel(associations.getCcdImageList(), sky_to_tan_projection,
+                                                   True, 0)
+
+        fit = lsst.jointcal.AstrometryFit(associations, model, self.config.posError)
+        fit.minimize("Distortions")
+        chi2 = fit.computeChi2()
+        self.log.info(str(chi2))
+        fit.minimize("Positions")
+        chi2 = fit.computeChi2()
+        self.log.info(str(chi2))
+        fit.minimize("Distortions Positions")
+        chi2 = fit.computeChi2()
+        self.log.info(str(chi2))
+
+        return fit, model
+
+    def _prep_astrometry_simplePolyModel(self, associations, sky_to_tan_projection):
+        model = lsst.jointcal.SimplePolyModel(associations.getCcdImageList(), sky_to_tan_projection,
+                                              True, 0, self.config.polyOrder)
+
+        fit = lsst.jointcal.AstrometryFit(associations, model, self.config.posError)
+        fit.minimize("Distortions")
+        chi2 = fit.computeChi2()
+        self.log.info(str(chi2))
+        fit.minimize("Positions")
+        chi2 = fit.computeChi2()
+        self.log.info(str(chi2))
+        fit.minimize("Distortions Positions")
+        chi2 = fit.computeChi2()
+        self.log.info(str(chi2))
+
+        return fit, model
+
```

Comment:
Good catch. Originally, `ConstrainedPolyModel` actually had a different "prep" sequence (the "Distortions"->"Positions"->"Distortions Positions" bit), but I reverted that after some testing. It could be different in the future, but I think for now it doesn't have to be, so I'll consolidate it as you suggest.